src/current/_includes/releases/v24.1/v24.1.0.md

```diff
@@ -241,7 +241,8 @@ In CockroachDB Self-Hosted, all available features are free to use unless their
 
 Before [upgrading to CockroachDB v24.1]({% link v24.1/upgrade-cockroach-version.md %}), be sure to review the following backward-incompatible changes, as well as [key cluster setting changes](#v24-1-0-cluster-settings), and adjust your deployment as necessary.
 
-- TBD
+- [`AS OF SYSTEM TIME`]({% link v23.2/as-of-system-time.md %}) queries can no longer use a timestamp followed by a question mark to signify a future-time value. This was an undocumented syntax. [#116830](https://github.com/cockroachdb/cockroach/pull/116830)
+- The `sql.txn.read_committed_isolation.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-setting.md %}) is now `true` by default. [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) transactions are no longer automatically upgraded to [`SERIALIZABLE`]({% link {{ page.version.version }}/demo-serializable.md %}), and will run as `READ COMMITTED` by default. As a result, upgrading CockroachDB may affect your application's behavior. To check whether any transaction is being upgraded from a weaker isolation level to `SERIALIZABLE` on v24.1, refer to the [**Upgrades of SQL Transaction Isolation Level**]({% link {{ page.version.version }}/ui-sql-dashboard.md %}#upgrades-of-sql-transaction-isolation-level) graph in the DB Console. If necessary, adjust those transactions before upgrading to v24.1. [#118479](https://github.com/cockroachdb/cockroach/pull/118479)
```

Comment:
I see that you changed the `v23.2` to `v24.1`, but this is incorrect. Citing `v23.2` is intentional because the user should check whether the automatic upgrade to `SERIALIZABLE` is happening for txns on v23.2. If those txns are being issued as `READ COMMITTED`, they will not be auto-upgraded on v24.1 and the app behavior will change. This is a bit nuanced and my original version wasn't clear enough. Maybe this is better?

```suggestion
- The `sql.txn.read_committed_isolation.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-setting.md %}) is now `true` by default. As a result, [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) transactions are **not** automatically upgraded to [`SERIALIZABLE`]({% link {{ page.version.version }}/demo-serializable.md %}), and will run as `READ COMMITTED` by default. On v23.2, refer to the [**Upgrades of SQL Transaction Isolation Level**]({% link {{ page.version.version }}/ui-sql-dashboard.md %}#upgrades-of-sql-transaction-isolation-level) graph in the DB Console to check whether any transaction is being upgraded from a weaker isolation level to `SERIALIZABLE`, and could therefore run differently on v24.1. [#118479](https://github.com/cockroachdb/cockroach/pull/118479)
```

Reply:
@rafiss Would also appreciate your eyes on this!